helm/flink-operator/templates/flink-operator.yaml

```diff
@@ -46,6 +46,10 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: ["/docker-entrypoint.sh", "operator"]
+          ports:
```

Comment:
Do we have to expose port by default ?  It's could be decided by a variable like `--set metric.port=9999` ? If not present, we can skip to declare, Right?

Reply:
Good idea, I'll make it optional

Reply:
Fixed